README.markdown

```diff
@@ -95,6 +95,46 @@ end
 
 See `lib/fakefs/spec_helpers.rb` for more info.
 
+### FakeFs vs `pp` --- `TypeError: superclass mismatch for class File`
+
+`pp` and `fakefs` collide, `require 'pp'` then `require 'fakefs'`.
+
+Mocking existing files
+----------------------
+
+If you want to use existing files in your fake fs, clone them first.
+
+```ruby
+it "can use existing files" do
+  FakeFS.activate!
+  FakeFS::FileSystem.clone('<folder-to-clone>')
+  expect(File.read('<folder-to-clone>/foo.yml')).to include("original-content-of-foo")
+end
+```
+
+For example, if you are running this from a spec folder, and would like to clone the cfg folder
```

Comment:
these 2 last paragraphs seem kind of redundant ... maybe make the first example be `.clone(File.expand_path('../../config', __FILE)) + File.read('../config/test.yml')` ... then add `File.write("../config/test.yml", "... and change it!")` ... and everything should be obvious :)